app/controllers/api_controller.rb

```diff
@@ -1,4 +1,5 @@
 class ApiController < ApplicationController
+  skip_before_filter  :verify_authenticity_token
```

Comment:
That's interesting. I think you are correct that we need to unify the auth methods. I'll take another look.